9.mAP/notebooks/0.generate_map_scores_class_level.ipynb

```diff
@@ -0,0 +1,954 @@
+{
```

Comment:
<p>Same comment about the P-values</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/pyroptosis_signature_data_analysis/pull/112/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>